### ISSUE TITLE
Incorrect nginx configuration

### ISSUE THREAD

**Issue by creitve** [simtechdev/smtk-devbox#16]
The server_name parameter and server root are wrong and this prevents any configuration with more than one site defined from working as intended.

**Comment by creitve** [comment]
```
# configuration file /etc/webkaos/conf.d/ailyin.u.simtech.conf:
server {
    listen 80;

    ############################################################################

    server_name ailyin.u.simtech;
    server_name ailyin.u.simtech
    charset utf-8;

    ############################################################################

    access_log /var/log/webkaos/u.simtech/ailyin.u.simtech/ailyin.u.simtech_access.log combined;
    error_log  /var/log/webkaos/u.simtech/ailyin.u.simtech/ailyin.u.simtech_error.log;

    ############################################################################

    root ailyin.u.simtech;

    ############################################################################

    # ssl                         on;
    # ssl_certificate             /etc/pki/tls/certs/ailyin.u.simtech.crt;
    # ssl_certificate_key         /etc/pki/tls/private/ailyin.u.simtech.key;
    # include                     xtra/ssl.conf;

    ############################################################################

    location @backend {
        proxy_pass http://127.0.0.1:8080;
        include xtra/proxy_params.conf;
        # include xtra/proxy_params_ssl.conf;
    }

    location / {
        try_files /maintenance.html @backend;

        location ~* \.(jpeg|ico|jpg|gif|png|css|js|pdf|txt|tar|wof|csv|zip) {
            access_log off;
            try_files $uri @backend;
            expires max;
            add_header Access-Control-Allow-Origin *;
            add_header Cache-Control public;
        }
    }

    ############################################################################
}

```

```
# configuration file /etc/webkaos/conf.d/landing_page.conf:
server {
    listen 80;

    ############################################################################

    server_name landing_page;
    server_name landing.ailyin.u.simtech
    charset utf-8;

    ############################################################################

    access_log /var/log/webkaos/landing_page/landing_page_access.log combined;
    error_log  /var/log/webkaos/landing_page/landing_page_error.log;

    ############################################################################

    root landing_page;

    ############################################################################

    # ssl                         on;
    # ssl_certificate             /etc/pki/tls/certs/landing_page.crt;
    # ssl_certificate_key         /etc/pki/tls/private/landing_page.key;
    # include                     xtra/ssl.conf;

    ############################################################################

    location @backend {
        proxy_pass http://127.0.0.1:8080;
        include xtra/proxy_params.conf;
        # include xtra/proxy_params_ssl.conf;
    }

    location / {
        try_files /maintenance.html @backend;

        location ~* \.(jpeg|ico|jpg|gif|png|css|js|pdf|txt|tar|wof|csv|zip) {
            access_log off;
            try_files $uri @backend;
            expires max;
            add_header Access-Control-Allow-Origin *;
            add_header Cache-Control public;
        }
    }

    ############################################################################
}

```

**Comment by gongled** [comment]
I've just fixed a bug and released version 2.4.0. 
@creitve could you try latest version of `smtk-devbox`? Please pull new changes and run `vagrant provision` again.
